randomize/randomize.go

```diff
@@ -52,7 +52,7 @@ var (
 	rgxValidTime     = regexp.MustCompile(`[2-9]+`)
 
 	validatedTypes = []string{
-		"inet", "line", "uuid", "interval", "mediumint",
+		"inet", "line", "uuid", "uniqueidentifier", "interval", "mediumint",
```

Comment:
I think this will add more problems than benefit. It's a bit of two different types of data, even though they look the same in the form of a string.

From Wiki:
> The term GUID usually refers to **Microsoft's implementation** of the Universally Unique Identifier (UUID) standard.